src/components/Footer/Footer-Types.ts

```diff
@@ -0,0 +1 @@
+export {};
```

Comment:
Empty object. If we don't have types for this component, we can delete this and created if it is needed.